charts/linkerd2/values.yaml

```diff
@@ -9,6 +9,9 @@ global:
   
   # Control Plane Trace Configuration
   controlPlaneTracing: false
+  
+  # Add custom host here
+  enforcedHostRegexp: ""
```

Comment:
`global` is only used for fields that are needed across the control-plane and add-ons (that are being added). So, This field can be moved outside.

Reply:
Thanks, Gotten